### ISSUE TITLE
Stand-alone sigmoid function?

### ISSUE THREAD

**Issue by shawnjhenry** [torch/torch7#499]
I'm working on some custom modules that make extensive use of the sigmoid function.  Some quick messing around seems to indicate that the implementation in nn.Sigmoid is substantially faster than any possible combination of torch.exp, torch.pow, torch.cdiv and ones tensors.  Obviously I could just use nn.Sigmoid as a function instead of a module, but that seems kind of cumbersome, and it would be much nicer to have a stand-alone torch.sigmoid function, especially since it's such a common use function. Would this be useful to anyone else?

Thanks,
Shawn

**Comment by fmassa** [comment]
This has been addressed in #500 and torch/cutorch#298
